ios/RCTTwilioChat/RCTTwilioChatClient.m

```diff
@@ -158,7 +158,7 @@ - (void)chatClient:(TwilioChatClient *)client connectionStateChanged:(TCHClientC
                                                  body:@(state)];
 }
 
-- (void)chatClient:(TwilioChatClient *)client synchronizationStatusChanged:(TCHClientSynchronizationStatus)status {
+- (void)chatClient:(TwilioChatClient *)client synchronizationStatusUpdated:(TCHClientSynchronizationStatus)status {
```

Comment:
@benoist this is the reason why the `synchronizationStatusChanged ` wasn't called.
Twilio changed the delegate call